aircan/dependencies/google_cloud/bigquery_handler_v2.py

```
@@ -0,0 +1,152 @@
+
+        }
+        e = replace_all(replacers,str(e))
+        logging.info(e)
+        status_dict = {
+            'res_id': ckan_conf.get('resource_id'),
+            'dag_run_id': ckan_conf.get('dag_run_id'),
+            'state': 'failed',
+            'message': str(e)
+        }
+        aircan_status_update(ckan_conf.get('site_url'), ckan_conf.get('api_key'), status_dict)
+        raise AirflowCKANException('Data ingestion has failed.', str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)